guides/source/active_record_migrations.md

```diff
@@ -246,7 +246,8 @@ class AddUserRefToProducts < ActiveRecord::Migration[5.0]
 end
 ```
 
-This migration will create a `user_id` column and appropriate index.
+This migration will create a `user_id` column and appropriate index. 
+For more index options visit the [API documentation](http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_index).
```

Comment:
Here we are discussing about `add_reference`, so we should point to add_reference documentation instead.